CONTRIBUTING.md

```diff
@@ -0,0 +1,89 @@
+# Contributing to awpy
+
+Hi! We're happy to have you here. Thank you in advance for your contribution to awpy.
+
+## Python code
+
+To make sure that awpy code stays readeable and works correctly we are making use of
```

Comment:
To make sure that awpy code stays readable and works correct, we are making use of a variety of helpful tools.